envoyextproc/response.go

```diff
@@ -0,0 +1,436 @@
+package envoyextproc
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"net/http"
+
+	ext_core_v3 "github.com/envoyproxy/go-control-plane/envoy/config/core/v3"
+	ext_proc_v3 "github.com/envoyproxy/go-control-plane/envoy/service/ext_proc/v3"
+	ext_type_v3 "github.com/envoyproxy/go-control-plane/envoy/type/v3"
+	"github.com/google/uuid"
+	"github.com/open-policy-agent/opa/v1/bundle"
+	"github.com/open-policy-agent/opa/v1/metrics"
+	"github.com/open-policy-agent/opa/v1/storage"
+	"github.com/open-policy-agent/opa/v1/topdown/builtins"
+	"google.golang.org/protobuf/types/known/structpb"
+)
+
+// EvalResult captures the result from evaluating a query against an input.
+type EvalResult struct {
+	Revision       string // Deprecated: Use `revisions` instead.
```

Comment:
We can drop this :) we don't need to preserve deprecated fields in a new implementation!